internal/controller/datadogmonitor/controller.go

```diff
@@ -92,11 +95,20 @@ func (r *Reconciler) internalReconcile(ctx context.Context, req reconcile.Reques
 	logger := r.log.WithValues("datadogmonitor", req.NamespacedName)
 	logger.Info("Reconciling DatadogMonitor")
 	now := metav1.NewTime(time.Now())
+	forceSyncPeriod := defaultForceSyncPeriod
+
+	forceSyncPeriodInt, err := strconv.Atoi(os.Getenv(DDMonitorForceSyncPeriodEnvVar))
+	if err != nil {
+		logger.Error(err, "Invalid value for monitor force sync period. Defaulting to 60 minutes.")
+	} else {
+		logger.V(1).Info("Setting monitor force sync period", "minutes", forceSyncPeriodInt)
```

Comment:
To avoid spamming the used sync period, this log is set to appear only in `debug` with `logger.V(1)`